### PR TITLE
ENH Add CUDA 11.1.1 support

### PR DESCRIPTION
**Pull request by mike-wendt**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This follows AnacondaRecipes/cudatoolkit-feedstock#13 which was for `11.1.0` but has not been reviewed yet

The idea is to get `11.1` out ASAP through `conda-forge` for those who need it, and Anaconda can copy this package into the `defaults` channel later

Given this is a different recipe approach from the PR above, I welcome feedback to know how to best contribute new versions in the future

cc @jakirkham @scopatz @raydouglass @kkraus14

### CODE REVIEW

File: README.md
Comment (mike-wendt) [inline README.md:35]: Not sure why this appeared other than maybe from running `conda smithy rerender` on my mac

If I need to drop this and rerender using the bot LMK

File: recipe/meta.yaml
Comment (mike-wendt) [inline recipe/meta.yaml:6]: I believe this should be `11.1` LMK if not

**Review by kkraus14 — APPROVED** [review]
Changes lgtm

@scopatz @jakirkham unsure how you want to handle the `major_minor` version in this recipe so defer to you for that detail

File: README.md
Comment (kkraus14) [reply README.md:35]: Will ask the bot to rerender once CI is done

**Comment by kkraus14** [comment]
@conda-forge-admin, please rerender

**Review by isuruf — CHANGES_REQUESTED** [review]
This looks incomplete.
See CI logs,
```
WARNING (cudatoolkit,lib/libaccinj64.so.11.1.105): $RPATH/libcupti.so.11.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
2020-11-05T19:47:11.2526306Z .. is this binary repackaging?
```

**Comment by mike-wendt** [comment]
> This looks incomplete.
> See CI logs,
> 
> ```
> WARNING (cudatoolkit,lib/libaccinj64.so.11.1.105): $RPATH/libcupti.so.11.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
> 2020-11-05T19:47:11.2526306Z .. is this binary repackaging?
> ```

I just checked the `11.0.3` [conda pkg](https://anaconda.org/conda-forge/cudatoolkit/11.0.3/download/linux-64/cudatoolkit-11.0.3-he1b5a44_0.tar.bz2) and it does not have `libcupti.so` either but I don't have the commit to reference to check the results to see if there is a warning there as well

`libcupti.so` is in a non-standard folder location so there will need to be a special copy command for it - I won't have time until tomorrow to try and extract this on a linux system to match the folders properly and to patch it. If anyone has a suggestion on the fix before then LMK

File: recipe/meta.yaml
Comment (jakirkham) [reply recipe/meta.yaml:6]: Yeah sorry this is leftover from other changes. Fixed with PR ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/24 ). Integrated that change into this PR as well. So this should be fixed.

**Comment by jakirkham** [comment]
> > This looks incomplete.
> > See CI logs,
> > ```
> > WARNING (cudatoolkit,lib/libaccinj64.so.11.1.105): $RPATH/libcupti.so.11.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
> > 2020-11-05T19:47:11.2526306Z .. is this binary repackaging?
> > ```
> 
> I just checked the `11.0.3` [conda pkg](https://anaconda.org/conda-forge/cudatoolkit/11.0.3/download/linux-64/cudatoolkit-11.0.3-he1b5a44_0.tar.bz2) and it does not have `libcupti.so` either but I don't have the commit to reference to check the results to see if there is a warning there as well

Just checked, it shows up in that build as well. Here's a [CI build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=232223&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d ) for reference and [its log file (for posterity)]( https://github.com/conda-forge/cudatoolkit-feedstock/files/5498109/linux_64_20201028-21.txt ).

**Comment by kkraus14** [comment]
Should we merge this in and address the issue(s) in a follow up before moving off of the `test` label?

**Comment by mike-wendt** [comment]
Closing this and will open a new PR once #31 merges

**Pull request closed without merging**
